### DISCUSSION TITLE
I can't use javascript...

### DISCUSSION THREAD

**Discussion by cfrikke** [3kh0/ext-remover#933] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
the url javascript:// is blocked, so please help

**Comment by AshtonDavies** [comment, marked as answer]
Can't bypass unless it's for LTBEEF and you're below v106 (highly unlikely situation). Policies are pretty much impossible to change without authority.

**Comment by cfrikke** [comment]
I got securly

**Comment by S-PScripts** [comment]
https://chromium-review.googlesource.com/c/chromium/src/+/4558108

**Comment by charlesluguda** [comment]
> the url javascript:// is blocked, so please help

It can be due to security issues. So try to use an inline script or external file script I think will help.

**Reply by AshtonDavies** [reply]
Bookmarklets are patched in v115 if devtools is disabled, which for you, it is. Not much you can do with them anymore anyway.
